### ISSUE TITLE
can't find pod spec

### ISSUE THREAD

**Issue by superjoo** [Juanpe/SkeletonView#27]
![screenshot](https://user-images.githubusercontent.com/1567417/34037473-7e8bf384-e14e-11e7-8ddb-78337e9d51a4.png)

when use "pod search skelentonView" in terminate, can't find pod spec.
Also pod install failed.

**Comment by Luur** [comment]
Hi, maybe because of typo
`pod search skeletonView`, not` pod search skeleNtonView`

**Comment by superjoo** [comment]
same as "pod search skeletonView "
![11111](https://user-images.githubusercontent.com/1567417/34086977-be78b73e-e365-11e7-8644-2722378c5861.png)

**Comment by Juanpe** [comment]
Hi @superjoo, could you try ```pod search SkeletonView```? The name of repository is "SkeletonView", is capitalized.

If not work with this, could you update your local pod repo? to do it, you only need to execute this command in the terminal: ```pod repo update```

**Comment by superjoo** [comment]
Hi @Juanpe ,also failed after excute `pod repo update`, and `pod search SkeletonView`.
Please see the screenshot.
![222](https://user-images.githubusercontent.com/1567417/34138118-919d241a-e433-11e7-92ae-ab667189e153.png)

**Comment by OmarBizreh** [comment]
I too was having the same issue but I replaced:
```
pod "SkeletonView"
```
with (don't know if that counts)
```
pod 'SkeletonView'
```

then ran: 
```
pod repo update
```

then: 
```
pod install
```
It installed: SkeletonView (1.0.4)

**Comment by Juanpe** [comment]
hi @superjoo, could you solve the problem? In order to close this issue 🙂

**Comment by Juanpe** [comment]
Solved! #18

**Comment by cellgit** [comment]
Solved:
![image](https://user-images.githubusercontent.com/24635226/60245654-9c654900-98ef-11e9-9cbf-3bf3ab50321f.png)
